backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -87,7 +87,11 @@ export const executeTransaction = async (
       transactionSend.typeId = TransactionTypeId.SEND
       transactionSend.memo = memo
       transactionSend.userId = sender.id
+      transactionSend.userGradidoID = sender.gradidoID
```

Comment:
Please test the new fields in both transactions

Reply:
do you mean the gradidoID too or only firstname and lastname?
The gradidoId is mandatory in the user-object.

firstname and lastname are checked now.